common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -1034,7 +1034,7 @@
   "feature.wynntils.tradeMarketBulkSell.bulkSell3Amount.description": "How many items should be sold when the bulk sell 3 button is pressed?",
   "feature.wynntils.tradeMarketBulkSell.bulkSell3Amount.name": "Bulk Sell 3 Amount",
   "feature.wynntils.tradeMarketBulkSell.description": "Adds bulk sell buttons in the trade market UI",
-  "feature.wynntils.tradeMarketBulkSell.name": "Trade Market Bulk Sell Buttons",
```

Comment:
I think we (and Wynntils) rather consistently refers to it as the "Trade Market", not just "Market". Is there any particular reason you want to change this?

Reply:
I agree that it is good to be consistent. But this is orthoginal to the silverbull tax issue, and this change is not the only one required to get to a consistent language. I recommend you revert these two changes, and we have a discussion in discord whether to use "Trade Market" or just "Market", and then open a PR where we fix all instances at once.